<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiChatDlg</class>
 <widget class="QDialog" name="MultiChatDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>收到消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="receiveMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="sendMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="contactList">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>151</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送(S)</string>
   </property>
  </widget>
  <widget class="QToolButton" name="icon_close">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="icon_mini">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
